modules/commons/src/main/java/org/apache/synapse/commons/json/JsonUtil.java

```diff
@@ -572,10 +598,9 @@ public static OMElement getNewJsonPayload(MessageContext messageContext, InputSt
                      * https://wso2.org/jira/browse/ESBJAVA-4270
                      */
                     if (isValidPayloadRequired(messageContext)) {
-                        logger.error(
-                                "#getNewJsonPayload. Could not save JSON payload. Invalid input stream found. MessageID: " +
-                                messageContext.getMessageID());
-                        throw new AxisFault("Payload is not a JSON string.");
+                        throw new AxisFault(
+                                "#getNewJsonPayload. Could not save JSON payload. Invalid input stream found. Payload" +
+                                        " is not a JSON string.");
```

Comment:
Lines should be wrapped before an operator.

Reply:
Fixed in #759